core/mock/loggerStub.go

```diff
@@ -2,28 +2,28 @@ package mock
 
 import "fmt"
 
-// LoggerMock -
-type LoggerMock struct {
+// LoggerStub -
+type LoggerStub struct {
```

Comment:
This is more than a mock rather than a stub. Suggestion to revert the rename and add a plain stub

Reply:
done